ios/RNIapIos.h

```diff
@@ -6,6 +6,8 @@
 {
   SKProductsRequest *productsRequest;
   NSMutableArray *validProducts;
+  SKPayment *promotedPayment;
+  SKProduct *promotedProduct;
```

Comment:
This is merged from the following code.

https://github.com/dooboolab/react-native-iap/blob/3f103f20c674c091fa69b715102bfc4caac37462/ios/IAPPromotionObserver.m#L4-L5